README.md

```diff
@@ -1,307 +1,62 @@
-EOF
-```
-**Apply the Cluster Role Binding file to the cluster:**
-```sh
-$ oc apply -f nacuser-rolebinding.yaml
-```
-
-**Alternatively Create Cluster Role Binding for our test user within nac-testing namespace:**
-```sh
-$ cat > nacuser-clusterrolebinding.yaml <<EOF
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: nacuser-nonadminbackup-cluster
-subjects:
-- kind: User
-  name: nacuser
-roleRef:
-  kind: ClusterRole
-  name: nonadminbackup-editor-role
-  apiGroup: rbac.authorization.k8s.io
-EOF
-```
-**Apply the Cluster Role Binding file to the cluster:**
-```sh
-$ oc apply -f nacuser-clusterrolebinding.yaml
-```
-
-### To Deploy on the cluster
-**Build and push your image to the location specified by `IMG`:**
-
-```sh
-# Example to build with podman
-export NAC_TAG="v0.0.1"
-export IMG_REGISTRY="quay.io/<USER>/oadp-nac"
-
-make docker-build docker-push IMG="${IMG_REGISTRY}:${NAC_REV}" CONTAINER_TOOL=podman
-```
-
-**NOTE:** This image ought to be published in the personal registry you specified.
-And it is required to have access to pull the image from the working environment.
-Make sure you have the proper permission to the registry if the above commands don’t work.
-
-**Install the CRDs into the cluster:**
-
-```sh
-make install
-```
-
-**Deploy the Manager to the cluster with the image specified by `IMG`:**
-
-```sh
-export NAC_TAG="v0.0.1"
-export IMG_REGISTRY="quay.io/<USER>/oadp-nac"
-
-make deploy IMG="${IMG_REGISTRY}:${NAC_REV}"
-```
-
-**Check your deployment**
-
-```sh
-oc get all -n oadp-nac-system
-```
-
-Result:
-```sh
-NAME                                               READY   STATUS    RESTARTS   AGE
-pod/oadp-nac-controller-manager-74bbf4577b-nssw4   2/2     Running   0          3m7s
-
-NAME                                                  TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
-service/oadp-nac-controller-manager-metrics-service   ClusterIP   172.30.201.185   <none>        8443/TCP   3m8s
-
-NAME                                          READY   UP-TO-DATE   AVAILABLE   AGE
-deployment.apps/oadp-nac-controller-manager   1/1     1            1           3m7s
-
-NAME                                                     DESIRED   CURRENT   READY   AGE
-replicaset.apps/oadp-nac-controller-manager-74bbf4577b   1         1         1       3m7s
-
-```
-
-> **NOTE**: If you encounter RBAC errors, you may need to grant yourself cluster-admin
-privileges or be logged in as admin.
-
-**Create instances of your solution**
-You can apply the samples (examples) from the config/sample:
-
-```sh
-kubectl apply -k config/samples/
-```
-
->**NOTE**: Ensure that the samples has default values to test it out.
+Non Admin Controller
 
-### To Uninstall
-**Delete the instances (CRs) from the cluster:**
+[![Continuos Integration](https://github.com/migtools/oadp-non-admin/actions/workflows/ci.yml/badge.svg)](https://github.com/migtools/oadp-non-admin/actions/workflows/ci.yml)
```

Comment:
Minor spelling edit, should be "Continuous". There are a few more in this pull request if you feel like fixing them.

Reply:
will change, thanks!

Reply:
done